meta-intel-edison-distro/recipes-support/blink-led/files/blink-led

```diff
@@ -49,18 +59,26 @@ def deinit_led_gpio():
     write_led(initial_led_state)
 
     # Deinit LED GPIO
-    subprocess.call("""
-    echo 40 >/sys/class/gpio/unexport
-    echo 214 >/sys/class/gpio/unexport
-    echo 243 >/sys/class/gpio/unexport
-    echo 261 >/sys/class/gpio/unexport
-    """, shell=True)
+    if acpi:
+        led.release()
+    else:
+        subprocess.call("""
+        echo 40 >/sys/class/gpio/unexport
+        echo 214 >/sys/class/gpio/unexport
+        echo 243 >/sys/class/gpio/unexport
+        echo 261 >/sys/class/gpio/unexport
+        """, shell=True)
 
 def signal_term_handler(signal, frame):
-    print 'Signal intercepted: de-initing LED GPIOs'
+    print('Signal intercepted: de-initing LED GPIOs')
```

Comment:
Throughout the script single and double-quoted strings are intermixed. I'd suggest to stick to one style or the other for uniformity.